roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -25,6 +25,7 @@ global
   daemon
   pidfile /var/run/haproxy/haproxy.pid
   tune.bufsize {{ haproxy.bufsize }}
+  stats socket /var/run/haproxy/stats.sock mode 600
```

Comment:
yessir, the socket is additive.  I could still access stats from the
http.   because config files are awesome that particular setting has to go
in global rather than default.

On Tue, Mar 31, 2015 at 12:43 PM, Dustin Lundquist <notifications@github.com
> wrote:

> In roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg
> <https://github.com/blueboxgroup/ursula/pull/911#discussion_r27503418>:
>
> > @@ -25,6 +25,7 @@ global
> >    daemon
> >    pidfile /var/run/haproxy/haproxy.pid
> >    tune.bufsize {{ haproxy.bufsize }}
> > +  stats socket /var/run/haproxy/stats.sock mode 600
>
> Have we verified this doesn't conflict with the HTTP stats in the default
> stanza below?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blueboxgroup/ursula/pull/911/files#r27503418>.
>